change/@microsoft-teams-js-4e6e7b05-3dd3-4b45-b74c-912281bbea04.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
Major isn't necessarily for breaking changes, I believe it also constitutes API changes. Though making Menus public I think would fall under minor.

Reply:
Thanks for the doc! I was hesitant between 'Major' and 'Minor' before as I was thinking of "Major: major feature; ..."(in our "beachball instruction"). If "MAJOR version means incompatible API changes, then I should change to Minor here.

Reply:
@erinha I would love your thoughts on this as we have been discussing versioning, etc.

Reply:
I agree. My reasoning is that this doesn't actually change the API surface. The APIs were exported from the SDK and app developers were able to call them and they haven't changed names or signatures. I think the biggest change for app developers is that the documentation will now be published. Another way of looking at it is that I wouldn't expect this change to cause us to release v3 vs. v2 (assuming we'd already GA'ed v2).

Reply:
FWIW, I think this is a bit of a strange scenario with respect to version bumping, because you kind of need to think about it in terms of what kind of change it was in v1. For example, the media API change is probably a minor change, the files API additions are also probably minor changes.

The simplest major change I can think of is breaking backwards compatibility. I'm not sure if we would consider a new capability to be a major change. This is something I need to think more about.

Reply:
Sounds good. Looks like we'll temporarily address this on a case by case basis.